src/app/pages/home/components/account-actions.tsx

```diff
@@ -42,13 +49,17 @@ export function AccountActions(props: FlexProps) {
           onClick={() => navigate(RouteUrls.FundChooseCurrency)}
         />
       )}
-
-      <ActionButton
-        data-testid={HomePageSelectors.SwapBtn}
-        icon={<SwapIcon />}
-        label="Swap"
-        onClick={() => navigate(RouteUrls.Swap)}
-      />
+      {whenNetwork(networkType)({
```

Comment:
Why would this only be based on the `networkType` for bitcoin?

Reply:
Yeah, a bit confusing in code if don't know both are always the same, will that always be the case with network refactoring?